crates/oxc_linter/Cargo.toml

```diff
@@ -36,6 +36,7 @@ oxc_syntax = { workspace = true }
 
 bitflags = { workspace = true }
 convert_case = { workspace = true }
+cow-utils = { workspace = true }
```

Comment:
Please run `taplo fmt`, I think our CI skips this.

Reply:
It's not skipped anymore!

Reply:
I ran `dprint fmt` locally again and didn't see any changes, so do I need to make changes here?

Reply:
No it's not needed.